src/equations/compressible_euler_1d.jl

```diff
@@ -68,11 +69,12 @@ A smooth initial condition used for convergence tests in combination with
 """
 function initial_condition_convergence_test(x, t,
                                             equations::CompressibleEulerEquations1D)
+    RealT = eltype(x)
     c = 2
-    A = 0.1
-    L = 2
+    A = convert(RealT, 0.1)
+    L = convert(RealT, 2) # since we divide by L below
```

Comment:
Because otherwise `x / 2` will always yield `Float64`? This should also be documented in the docs (but thanks for the comment here!)

Reply:
Yes